torchvision/utils.py

```diff
@@ -320,6 +320,8 @@ def draw_bounding_boxes(
         font_size (int): The requested font size in points.
         label_colors (color or list of colors, optional): Colors for the label text.  See the description of the
             `colors` argument for details.  Defaults to the same colors used for the boxes, or to black if ``fill_labels`` is True.
+        label_background_colors (color or list of colors, optional): Colors for the label text box fill. Defaults to the
+            same colors used for the boxes. Ignored when ``fill_labels`` is False.
```

Comment:
Do we really ignore the `label_background_colors` when ``fill_labels`` is False?

Reply:
No I think you were right initially, let's keep it like that!